crates/pathfinder/examples/dump_events.rs

```diff
@@ -58,16 +65,29 @@ fn make_ground_path(cli: &Cli) -> anyhow::Result<PathBuf> {
     } else {
         format!("{}+{}", cli.from_block, cli.to_block - cli.from_block)
     };
+
+    let filter_number = 0;
```

Comment:
What exactly is this `filter_number` here? It seems it's always zero?

Reply:
Well, basically it's the part of the test input (and corresponding filter spec) names that distinguishes them from other filter specs and test inputs - e.g. the `9` in `0+1000f9.json` and corresponding `0+1000w9.jsonl` - so in principle it could be any string, but existing test generators (like https://github.com/eqlabs/starknet-event-query/blob/main/examples/filter_address.rs ) only use numbers there (starting from 1), and so when I implemented direct filtering in `dump_events` (which only produces 1 filter spec + expected result) I went for 0... Maybe it should be made more structured and configurable, so that one could generate more filtered groups from a single block range, but so far that seemed too complex... :-)